ads/_config.js

```diff
@@ -402,6 +402,11 @@ export const adConfig = {
 
   engageya: {},
 
+  epeex: {
+    prefetch: 'https://epeex.com/related/service/widget/amp/remote.js',
+    renderStartImplemented: true,
```

Comment:
I am not seeing this fire. Also we do not require this for `amp-embed` but you are welcome to add it for something like request to resize.